src/osemgrep/networking/Rule_fetching.ml

```diff
@@ -425,13 +426,13 @@ let rules_from_dashdash_config_async ~rewrite_rule_ids ~token_opt
       |> Lwt.return
   | C.URL url ->
       let%lwt rules =
-        load_rules_from_url_async ~origin:(Untrusted_remote url) url
+        load_rules_from_url_async ~origin:(Untrusted_remote url) ~token_opt url
```

Comment:
Does it really make sense to send the token here?

Reply:
Fair question. I think for consistency, yes?

Reply:
Wait a sec – this is the bug I was fixing!